airflow/secrets/local_filesystem.py

```diff
@@ -145,6 +144,7 @@ def _parse_json_file(file_path: str) -> Tuple[Dict[str, Any], List[FileSyntaxErr
     "env": _parse_env_file,
     "json": _parse_json_file,
     "yaml": _parse_yaml_file,
+    "yml": _parse_yaml_file,
```

Comment:
This is the only behavior-changing line. It has the following effects:
- one can import connections from `.yml` files via the command-line
- `LocalFilesystemBackend` can now reference variables stored in `.yml` files